### PR TITLE
Api

### PR DESCRIPTION
**Pull request by rokoc003**
image

### CODE REVIEW

File: src/components/pokemons.js [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
-  props.items.forEach(item => {
+  props.items.forEach((item, pokemonId) => {
     const element = createElement('div', {
       innerText: item,
       className: 'pokemon'
     });
+    const pokemonImage = createElement('img', {
+      className: 'pokemonImage',
+      src: `https://pokeres.bastionbot.org/images/pokemon/${pokemonId + 1}.png`
+    });
```
Comment (rokoc003) [inline src/components/pokemons.js]: image

Just added the index as pokemonId to change the number of each pokemon

**Comment by Kathrin92** [comment]
lgtm 👍really nice styling :)

File: src/components/pokemons.js [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
 
-  props.items.forEach(item => {
+  props.items.forEach((item, pokemonId) => {
     const element = createElement('div', {
       innerText: item,
       className: 'pokemon'
     });
+    const pokemonImage = createElement('img', {
+      className: 'pokemonImage',
+      src: `https://pokeres.bastionbot.org/images/pokemon/${pokemonId + 1}.png`
```
Comment (marwinburesch) [inline src/components/pokemons.js]: don't forget the `alt` attribute :)
```suggestion
      src: `https://pokeres.bastionbot.org/images/pokemon/${pokemonId + 1}.png`,
      alt: `image of pokemon #${pokemonId}`
```

File: src/index.scss [hunk 3]
```diff
@@ -14,7 +14,7 @@ body {
     theme.$background-color-light,
     theme.$background-color-dark
   );
-  height: 100vh;
+  height: 1000vh;
```
Comment (marwinburesch) [inline src/index.scss]: `1000vh`? 🧐

**Pull request merged**